board.py

```diff
@@ -112,141 +109,140 @@ def getTilePositions(self, position):
-                "Active Side: " + ("Side 1" if self.activeSide == self.side1 else "Side 2") + "\n" + \
-                "Orientation: " + str(self.orientation) + "\n"
+        return "Side 1: " + str(self.side1) + "\n" + \
+               "Side 2: " + str(self.side2) + "\n" + \
+               "Rotation Code: " + str(self.rotationCode) + "\n" + \
+               "Active Side: " + ("Side 1" if self.activeSide == self.side1 else "Side 2") + "\n" + \
+               "Orientation: " + str(self.orientation) + "\n"
 
 
 class Board:
     DIMENSIONS_X_Y = (8, 12)
-    CONVERSION_LETTER_TO_NUMBER = {'A': 0, 'B': 1, 'C': 2, 'D': 3, 'E': 4, 'F': 5, 'G': 6, 'H': 7, 'I': 8,
-                                'J': 9, 'K': 10, 'L': 11, 'M': 12, 'N': 13, 'O': 14, 'P': 15, 'R': 16, 'S': 17,
-                                'T': 18, 'U': 19, 'V': 20, 'W': 21, 'X': 22, 'Y': 23, 'Z': 24}
+    CONVERSION_LETTER_TO_NUMBER = {'A': 0, 'B': 1, 'C': 2, 'D': 3, 'E': 4, 'F': 5, 'G': 6, 'H': 7}
 
-    def __init__(self, maxNbrCards):
+    def __init__(self, max_nbr_cards):
         # NOTE: Initially, the board is empty (no cards on it), so no tiles are on it either.
         # We illustrate a location on the board with no tile/card as a string (blank spaces).
-        self.board = [[' ' * 4 for x in range(self.DIMENSIONS_X_Y[0])] for y in range (self.DIMENSIONS_X_Y[1])]
+        self.board = [[' ' * 4 for x in range(self.DIMENSIONS_X_Y[0])] for y in range(self.DIMENSIONS_X_Y[1])]
         self.nbrCards = 0
         # There is at most maxNbrCards cards on the board and we have to ensure no one can insert cards (through the
         # methods we provide) when we reach that quota.
-        self.maxNbrCards = maxNbrCards
+        self.maxNbrCards = max_nbr_cards
 
-    def convertCoordinate(self, letterNumberCoord):
+    def convert_coordinate(self, letter_num_coordinate):
         """ Convert a coordinate in the form [A-Z] [0-9] (eg. A 2) (given as a tuple)
             to a coordinate in the current 2-dimensional array
             Note the letter is the column and the number is the row (so coordinate given as (column, row))
         """
-        xCoord = self.convertLetterToNumber(letterNumberCoord[0])
-        if xCoord >= self.DIMENSIONS_X_Y[0]:
-            raise Exception("ERROR: X coordinate " + str(letterNumberCoord[0]) + " is out of bounds")
-        yCoord = letterNumberCoord[1] - 1
-        if yCoord >= self.DIMENSIONS_X_Y[1] or yCoord < 0:
-            raise Exception("ERROR: Y coordinate " + str(letterNumberCoord[1]) + " is out of bounds")
-        return (xCoord, yCoord)
-
-    def convertLetterToNumber(self, letter):
+        x_coord = self.convert_letter_to_num(letter_num_coordinate[0])
+        if x_coord >= self.DIMENSIONS_X_Y[0]:
+            raise OutOfBoundsException("ERROR: X coordinate " + str(letter_num_coordinate[0]) + " is out of bounds")
+        y_coord = letter_num_coordinate[1] - 1
+        if y_coord >= self.DIMENSIONS_X_Y[1] or y_coord < 0:
```

Comment:
This is the not hardcoded version of the hardcoded check below
if searched_number < 0 or searched_number > 7: